docs/manage/CLI.md

```diff
@@ -347,6 +347,10 @@ viam data export --destination=/home/robot/data --data_type=tabular \
 # export binary data from all orgs and locations, component name myComponent
 viam data export --destination=/home/robot/data --data-type=binary \
 --component-name myComponent
+
+# configure a new database user for use with data query
```

Comment:
I would specify that this is for tabular data, and that this is a database user for the Viam organization's MongoDB Atlas Data Federation instance